glib/gmain_context.go

```diff
@@ -25,6 +25,16 @@ func MainContextDefault() *MainContext {
 	return (*MainContext)(c)
 }
 
+// Iteration is a wrapper around g_main_context_iteration()
+func Iteration(context *MainContext, mayBlock bool) bool {
+	return gobool(C.g_main_context_iteration(context.native(), gbool(mayBlock)))
+}
+
+// Pending is a wrapper around g_main_context_pending()
+func Pending(context *MainContext) bool {
+	return gobool(C.g_main_context_pending(context.native()))
+}
+
```

Comment:
Shouldn't these be `context.Iteration(bool)` and `context.Pending()` instead of `Iteration(context, bool)` and `Pending(context)`?

Reply:
indeed